Userland/Libraries/LibPDF/ObjectDerivatives.h

```diff
@@ -113,6 +113,9 @@ class DictObject final : public Object {
     template<typename... Args>
     bool contains(Args&&... keys) const { return (m_map.contains(keys) && ...); }
 
+    template<typename... Args>
+    bool contains_any_of(Args&&... keys) const { return (m_map.contains(keys) || ...); }
```

Comment:
Might be a good idea to rename `contains` to `contains_all_of`, to be more explicit